padd.sh

```diff
@@ -1387,6 +1387,34 @@ NormalPADD() {
   done
 }
 
+Update() {
+    GetPADDInformation
```

Comment:
How should we handle PADD on the Docker image? I think it does not make sense to allow an update on the docker container. Therefore you need to exit if on docker. Check `ShowVersion` how to handle docker.

Reply:
In `GetPADDInformation` there already is a check for running on Docker, which is why I omitted another check in `Update` and only check for `padd_out_of_date_flag` being true - which right now can only happen if we are not running in Docker (and an update exists).

Reply:
You need to source the versions file first.

https://github.com/pi-hole/PADD/blob/3c2da235e8e0849a9ac37639d9bf305e8f032284/padd.sh#L1199-L1200

Reply:
Done 👍🏼